clang/lib/Basic/Targets/AMDGPU.cpp

```diff
@@ -32,9 +32,9 @@ static const char *const DataLayoutStringR600 =
     "-v192:256-v256:256-v512:512-v1024:1024-v2048:2048-n32:64-S32-A5-G1";
 
 static const char *const DataLayoutStringAMDGCN =
-    "e-p:64:64-p1:64:64-p2:32:32-p3:32:32-p4:64:64-p5:32:32-p6:32:32"
-    "-p7:160:256:256:32-p8:128:128-p9:192:256:256:32-i64:64-v16:16-v24:32-v32:"
-    "32-v48:64-v96:128"
+    "e-p:64:64-p1:64:64-p2:32:32-p3:32:32:32:32:f-p4:64:64-p5:32:32:32:32:f"
+    "-p6:32:32-p7:160:256:256:32-p8:128:128-p9:192:256:256:32-i64:64-v16:16"
+    "-v24:32-v32:32-v48:64-v96:128"
```

Comment:
Not sure about the interaction with fat pointers. It only applies to a subset of the bits. But also not that important, we probably can't do much with that information

Reply:
Right, that's why I didn't try to add that information to the fat pointer at the moment, and it is all-zero anyway.